.travis/prerelease_lobby_install

```diff
@@ -1,39 +0,0 @@
-#!/bin/bash
```

Comment:
We probably want to readd a similar script in the future in order to test compatibility,
But we should probably use start-stop-daemon instead of nohup then as suggested by the [travis docs](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI)